Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/Base32.kt

```diff
@@ -0,0 +1,25 @@
+package de.rki.coronawarnapp.eventregistration
+
+import com.google.common.io.BaseEncoding
+
+/**
+ * Decodes input String into byte array using Base32 decoder
+ * @return [ByteArray]
+ */
+fun String.decodeBase32ToArray(): ByteArray = BaseEncoding.base32().decode(this)
```

Comment:
What do you think about decoding to `ByteString` (`decodeBase64()` also uses it), less custom code from our end, and the API consumer of these extensions has more options.

Reply:
`decodeBase32(): ByteString`

Reply:
Good idea, Added a new variant for it. we can use what suits the use case.
`decodeBase32ToByteString`